### DISCUSSION TITLE
How to assign a person full access to a single kargo project only

### DISCUSSION THREAD

**Discussion by creeram** [akuity/kargo#3021] [General]
<img width="1417" alt="Screenshot 2024-11-29 at 7 10 32 PM" src="https://github.com/user-attachments/assets/1205d10d-62c4-4e64-b408-dc6414fd9c6f">

I tried to create a role and assign it to an email claim but the user cannot list the projects. you can see it in the above screenshot for the error message. How can I make it work?

Steps to create and assign role.

```
kargo create role developer --project kargo-demo

kargo grant --role developer \                                           
  --verb '*' --resource-type stages \
  --project kargo-demo
  
kargo grant --role developer --claim email=user@example.com --project kargo-demo
```

**Comment by krancour** [comment]
Closing this as the question was answered in #3023.
